### PR TITLE
Dev milestone1 - Calculator

### PR DESCRIPTION
**Pull request by Wusinho**
This project is an App Calculator using React
Built with:

- HTML
- Bootstrap
- JavaScript
- Webpack
- Package.json
- React

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Todo-List
```
Comment (rindrajosia) [inline README.md]: Please change it to CALCULATOR.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- `big.js` is added to package.json :100: 
- `prop-types` is added to package.json :100: 
- app runs correctly with `npm start`

To improve :factory_worker: [REQUIRED]:

- [ ] This APP must be initialized with `create-react-app` ([create-react-app and Delete any unnecessary file or asset screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/useCreateAppAndDeleteUnusedFiles.png))
- [ ] Unnecessary files or assets should be deleted like dist etc, ...
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

- [ ] Please re-check your readme as it is the first thing users read. For example, the title of your project is Todo-List

Nice to have [OPTIONAL] :

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 

- ### To Improve
  - [x] [MANDATORY] Most of the issues raised in the previours review are not addressed. Kindly refer to the previous review and fix them.
    - The App must be initialized with `create-react-app` ([create-react-app and Delete any unnecessary file or asset screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/useCreateAppAndDeleteUnusedFiles.png))
    - With the app initialized by `create-react-app`, we don't need to configure webpack manually.

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

**Comment by Wusinho** [comment]
React is working do I need to make a new repo with create-react-app ???

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED 🎉 
@Wusinho well done on this milestone. kindly merge prepare for the next task

**Pull request merged**